### PR TITLE
[Buckinghamshire] Alloy configuration

### PR DESCRIPTION
**Pull request by davea**
[skip changelog]

### CODE REVIEW

File: perllib/FixMyStreet/Cobrand/Buckinghamshire.pm [hunk 1]
```diff
@@ -110,6 +111,34 @@ sub send_questionnaires {
 
 sub open311_extra_data_exclude { [ 'road-placement' ] }
 
+sub open311_extra_data_include {
+    my ($self, $row, $h) = @_;
+
+    my $open311_only = [
```
Comment (dracos) [inline perllib/FixMyStreet/Cobrand/Buckinghamshire.pm:117]: Worth calling super here (or using around)? Doesn't really matter, unlikely we add a new central Alloy thing is it.

**Review by dracos — CHANGES_REQUESTED** [review]
Nice additions to the cobrand features :) All looks good - would be great to have a test on the new look up site code stuff (think the current test mocks a bit widely, perhaps), and some docs on the new things :)

File: perllib/FixMyStreet/Cobrand/Buckinghamshire.pm (hunk 1 above)
Comment (davea) [reply perllib/FixMyStreet/Cobrand/Buckinghamshire.pm]: `around` is a good shout 👍

**Review by dracos — APPROVED** [review]
Thanks, looks good :) Don't understand it saying some lines in Buckinghamshire.pm aren't covered, they must be surely as I'm reading it. Ah well!

**Pull request merged**